workplace-incoming-hook.go

```diff
@@ -399,7 +399,11 @@ func MergeHandler(body string) {
 			j.ObjectAttributes.Source.Name,
 			j.ObjectAttributes.SourceBranch,
 			dateString) + n
-		// Third line (last commit message)
+
+		// Second line (URL link for merge request location)
+		message += fmt.Sprintf("Link : *%s*", j.ObjectAttributes.URL) + n
```

Comment:
can we see the message screenshot for this line ?

Reply:
![image](https://user-images.githubusercontent.com/5858756/61168099-7b009000-a572-11e9-902e-4e58b9f696a3.png)

*note : this mr with empty description